MantidQt/CustomInterfaces/src/Tomography/TomographyIfacePresenter.cpp

```diff
@@ -134,7 +150,10 @@ void TomographyIfacePresenter::notify(
 }
 
 void TomographyIfacePresenter::processSystemSettingsUpdated() {
-  m_model->updateSystemSettings(m_view->systemSettings());
+  m_model->setSystemSettings(m_view->systemSettings());
+
+  // update the tool information with the new system settings
+  setupConfigDialogSettingsAndUpdateModel(m_configDialog.get());
```

Comment:
I prefer passing just the pointer so that way the methods don't need to care about how we're storing the pointer